translations/fr.json

```diff
@@ -342,218 +333,209 @@
   "actionbox.city": "La cité de {0}",
   "actionbox.city.level": "niveau {0}",
   "actionbox.village": "Village",
-  "actionbox.unit.frozen": "{0} glacés",
-  "actionbox.unit.kills": "{0}/{1} morts",
+  "actionbox.unit.frozen": "Glacé {0}",
+  "actionbox.unit.kills": "{0}/{1} Morts",
   "actionbox.unit.veteran": "Vétéran",
   "actionbox.unit.new": "Entraîner un nouveau {0} {1}",
-  "actionbox.unit.ability": "Capacité d'unité",
+  "actionbox.unit.ability": "Capacité",
   "actionbox.unit.train": "ENTRAÎNER",
   "actionbox.unit.upgrade": "AMÉLIORER",
-  "actionbox.unit.toomany": "(trop nombreux)",
-  "actionbox.unit.toomany.info": "Cette ville ne peut plus contenir d'unités. Améliorez votre ville pour avoir plus d'espace pour les unités.",
-  "actionbox.building.doit": "LE FAIRE",
-  "actionbox.building.requiredtech": "Vous avez besoin de rechercher {0} pour faire ça, cliquez sur le bouton 'Recherche' .",
-  "actionbox.building.techtree": "RECHERCHES",
-  "actionbox.insufficientfunds": "Vous n'avez pas assez d'étoiles pour l'acheter. Appuyez sur 'Prochain tour' pour avoir plus d'étoiles.",
-  "actionbox.confirm": "Confirmez {0}",
+  "actionbox.unit.toomany": "(Trop nombreux)",
+  "actionbox.unit.toomany.info": "Cette villes ne peut plus contenir d'unités. Améliorer votre ville pour pouvoir en refaire",
+  "actionbox.building.doit": "APPLIQUER",
+  "actionbox.building.requiredtech": "Vous avez besoin de rechercher '{0}' pour faire ça, appuyer sur le boutton 'Arbre Technologique' .",
+  "actionbox.building.techtree": "Arbre Technologique",
+  "actionbox.insufficientfunds": "Vous n'avez pas assez d'étoiles pour acheter ça. Appuyer sur 'Tour suivant' pour avoir plus d'étoiles.",
+  "actionbox.confirm": "Confirmer {0}",
   "actionbox.confirm.info": "Êtes vous sûr de vouloir faire ça ?",
-  "actionbox.confirm.button": "OUI",
-
+  "actionbox.confirm.button": "oui",
   "tooltip.tile.road": "Construisez une route pour connecter cette ville à votre capitale.",
-  "tooltip.tile.choose_unit": "Choisissez une unité à produire.",
-  "tooltip.tile.limit": "Cette ville ne peut plus faire d'autre unité.",
-  "tooltip.tile.capture.enemy": "Votre ville est en train d'être capturée par l'ennemi !",
+  "tooltip.tile.choose_unit": "Choissisez une unité à produire.",
+  "tooltip.tile.limit": "Cette ville ne peut plus créer d'autres unités.",
+  "tooltip.tile.capture.enemy": "Votre ville est en train d'être capturé par l'ennemi !",
   "tooltip.tile.capture": "Cette ville est en train d'être capturée.",
-  "tooltip.tile.capture.tip": "Déplacez une unité ici pour capturer cette ville !",
+  "tooltip.tile.capture.tip": "Bouger une unité içi pour capturé cette ville !",
```

Comment:
Move means "Bouger" but in this case, "deplacer" have more sense. And obviously, "capturer" is not a past participle.... The worst is a "ç" in "ici".... Why?